packages/web/app/package.json

```
@@ -5,11 +5,11 @@
   "scripts": {
     "build": "tsx ../../../scripts/runify.ts src/server/index.ts && vite build --outDir dist/client",
     "build-storybook": "storybook build",
-    "dev": "node --env-file=.env --watch-path src/server --import tsx src/server/index.ts --dev",
+    "dev": "tsx watch --clear-screen=false --exclude \"./**/*.mjs\" src/server/dev.ts",
```

```suggestion
    "dev": "NODE_ENV=development tsx watch --clear-screen=false --exclude \"./**/*.mjs\" src/server/dev.ts",
```
is this will works? instead of `process.env.NODE_ENV = 'development';` in `server/dev.ts`

---

Why though? It won’t work on Windows and it’s just odd to pass env vars like that for that single purpose.

---

just to avoid creating yet another file for dev environment? we can pass this env via `cross-env` package to be platform agnostic

---

200 files or 180 makes zero difference